lib/get-chaged-files.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+var minimatch = require('minimatch');
+var reduce = require('lodash.reduce');
+
+/**
+ * Returns an array of changed files comparing current timestamps
+ * against cached timestamps from previous run.
+ *
+ * @param plugin - stylelint-webpack-plugin this scopr
+ * @param compilation
```

Comment:
Do we need the entire `compilation` object here? If I haven't misread the code looks like we're just using the `fileTimestamps` property so I would just accept that as a param.